packages/suite/src/actions/wallet/coinjoinClientActions.ts

```diff
@@ -75,6 +75,42 @@ export type CoinjoinClientAction =
     | ReturnType<typeof clientSessionRoundChanged>
     | ReturnType<typeof clientSessionCompleted>;
 
+const setBusyScreen =
+    (accountKeys: string[], expiry?: number) => (_dispatch: Dispatch, getState: GetState) => {
+        const {
+            devices,
+            wallet: { accounts },
+        } = getState();
+
+        const groupDeviceState = accountKeys.flatMap(key => {
```

Comment:
nit: sounds like a function name

Reply:
so whats your proposal then?

Reply:
`accountDeviceStates` but I see you have already changed it! 👌